frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-select-template.tsx

```diff
@@ -0,0 +1,280 @@
+                <FilterSidePanelCategory key="source" title={BootSourcesTitle}>
+                  {[BOOT_SOURCE_COMMUNITY, BOOT_SOURCE_USER, BOOT_SOURCE_REQUIRED].map((source) => (
+                    <FilterSidePanelCategoryItem key={source} {...getSourceItemProps(source)}>
+                      {source}
+                    </FilterSidePanelCategoryItem>
+                  ))}
+                </FilterSidePanelCategory>
+              </FilterSidePanel>
+            </div>
+            <div className="co-catalog-page__content">
+              <Stack hasGutter>
+                <StackItem>
+                  <div className="co-catalog-page__header">
+                    <Split hasGutter>
+                      <SplitItem isFilled>
+                        <InputGroup>
+                          <TextInput
+                            name="textFilter"
+                            id="textFilter"
+                            type="search"
+                            aria-label="text filter"
+                            onChange={(text) => onSelect('text', text)}
+                            value={filters.text || ''}
+                            placeholder="Search by name, OS ..."
+                          />
+                          <Button variant={ButtonVariant.control} aria-label="Search">
+                            <SearchIcon />
+                          </Button>
+                        </InputGroup>
+                      </SplitItem>
+                      <SplitItem isFilled />
+                      <SplitItem className="co-catalog-page__num-items">
+                        {pluralize(filteredItems.length, 'item')}
+                      </SplitItem>
+                    </Split>
+                  </div>
+                </StackItem>
+                <StackItem>
+                  <StatusBox
+                    data={templates}
+                    loaded={loaded}
+                    loadError={loadError}
+                    label="Templates"
+                  >
+                    {filteredItems?.length > 0 ? (
+                      <VirtualizedGrid
+                        className="co-catalog-page__grid"
+                        items={filteredItems}
+                        renderCell={renderTile}
+                        cellWidth={300}
+                        celldefaultHeight={340}
+                      />
+                    ) : (
+                      <EmptyState>
+                        <Title headingLevel="h4" size="lg">
+                          No Results Match the Filter Criteria
+                        </Title>
+                        <EmptyStateBody>
+                          No Virtual Machine templates items are being shown due to the filters
+                          being applied.
+                        </EmptyStateBody>
+                        <EmptyStateSecondaryActions>
+                          <Button variant="link" onClick={() => clearFilter()}>
```

Comment:
```suggestion
                          <Button variant="link" onClick={clearFilter}>
```

Reply:
clearfilter can be empty or have args, but cant handle `e`